src/server/wesnothd/game.hpp

```diff
@@ -117,199 +180,352 @@ class game
 		return &data;
 	}
 
+	/**
+	 * @return The nodes containing the sides in this game.
+	 */
 	const simple_wml::node::child_list& get_sides_list() const
 	{
 		return starting_pos(level_.root())->children("side");
 	}
 
+	/**
+	 * @return Whether this game has started yet.
+	 */
 	bool started() const
 	{
 		return started_;
 	}
 
+	/**
+	 * @return The number of players. One player can have multiple sides.
+	 */
 	std::size_t nplayers() const
 	{
 		return players_.size();
 	}
 
+	/**
+	 * @return The number of observers in this game.
+	 */
 	std::size_t nobservers() const
 	{
 		return observers_.size();
 	}
 
+	/**
+	 * @return This game's current turn.
+	 */
 	std::size_t current_turn() const
 	{
 		return current_turn_;
 	}
 
-	void set_current_turn(int turn)
-	{
-		current_turn_ = turn;
-	}
-
+	/**
+	 * @return The name of the replay for this game.
+	 */
 	std::string get_replay_filename();
 
+	/** Toggles whether all observers are muted or not. */
 	void mute_all_observers();
 
 	/**
-	 * Mute an observer or give a message of all currently muted observers if no
-	 * name is given.
+	 * Mute an observer or give a message of all currently muted observers if no name is given.
+	 * @todo FIXME: Maybe rather save muted nicks as a set of strings and also allow muting of usernames not in the game.
```

Comment:
Sure, same thing for github issues though. Some people will look at the code some will look at the bug tracker for things to do.

For the people that look at the code I think it's more likely they will look at the actual code that the TODO is about instead of the header.